invenio_app_ils/circulation/search.py

```diff
@@ -105,6 +106,16 @@ def get_most_loaned_documents(self, from_date, to_date, bucket_size):
 
         return search
 
+    def get_all_overdue_loans(self):
+        """Return all loans that are overdue."""
+        search = current_circulation.loan_search
+        states = current_app.config["CIRCULATION_STATES_LOAN_ACTIVE"]
+        search = search.query("bool", must=[
+            Q("terms", state=states),
+            Q("range", end_date=dict(lte=datetime.utcnow())),
```

Comment:
Shouldn't it be `lt` instead of `lte` otherwise it is considered overdue if the `end_date` is today?